Modules/Core/Common/include/itkMultiThreaderBase.h

```diff
@@ -61,60 +61,75 @@ class ProcessObject;
 class ITKCommon_EXPORT MultiThreaderBase : public Object
 {
 public:
-  ITK_DISALLOW_COPY_AND_ASSIGN(MultiThreaderBase);
+  ITK_DISALLOW_COPY_AND_ASSIGN( MultiThreaderBase );
 
   /** Standard class type aliases. */
   using Self = MultiThreaderBase;
   using Superclass = Object;
-  using Pointer = SmartPointer<Self>;
-  using ConstPointer = SmartPointer<const Self>;
+  using Pointer = SmartPointer< Self >;
+  using ConstPointer = SmartPointer< const Self >;
 
   /** Method for creation through the object factory. */
-  static Pointer New();
+  static Pointer
+  New();
 
   /** Run-time type information (and related methods). */
-  itkTypeMacro(MultiThreaderBase, Object);
+  itkTypeMacro( MultiThreaderBase, Object );
 
   /** Get/Set the number of threads to use. It will be clamped to the range
    * [ 1, m_GlobalMaximumNumberOfThreads ], so the caller of this method should
    * check that the requested number of threads was accepted. */
-  virtual void SetMaximumNumberOfThreads( ThreadIdType numberOfThreads );
+  virtual void
+  SetMaximumNumberOfThreads( ThreadIdType numberOfThreads );
   itkGetConstMacro( MaximumNumberOfThreads, ThreadIdType );
 
   /** Get/Set the number of work units to create. It might be clamped to the range
    * [ 1, SomeMaximumNumber ], so the caller of this method should
    * check that the requested number of work units was accepted. */
-  virtual void SetNumberOfWorkUnits( ThreadIdType numberOfWorkUnits );
+  virtual void
+  SetNumberOfWorkUnits( ThreadIdType numberOfWorkUnits );
   itkGetConstMacro( NumberOfWorkUnits, ThreadIdType );
 
   /** Set/Get the maximum number of threads to use when multithreading.  It
    * will be clamped to the range [ 1, ITK_MAX_THREADS ] because several arrays
    * are already statically allocated using the ITK_MAX_THREADS number.
    * Therefore the caller of this method should check that the requested number
    * of threads was accepted. */
-  static void SetGlobalMaximumNumberOfThreads(ThreadIdType val);
-  static ThreadIdType GetGlobalMaximumNumberOfThreads();
+  static void
+  SetGlobalMaximumNumberOfThreads( ThreadIdType val );
+  static ThreadIdType
+  GetGlobalMaximumNumberOfThreads();
 
   /** Set/Get whether to use the to use the thread pool
    * implementation or the spawing implementation of
    * starting threads.
    *
    * Deprecated: use Get/Set GlobalDefaultThreader. */
-  itkLegacyMacro(static void SetGlobalDefaultUseThreadPool( const bool GlobalDefaultUseThreadPool ));
-  itkLegacyMacro(static bool GetGlobalDefaultUseThreadPool( ));
+  itkLegacyMacro( static void SetGlobalDefaultUseThreadPool( const bool GlobalDefaultUseThreadPool ) );
+  itkLegacyMacro( static bool GetGlobalDefaultUseThreadPool() );
 
   /** Currently supported types of multi-threader implementations.
    * Last will change with additional implementations. */
-  enum ThreaderType { Platform = 0, First = Platform, Pool, TBB, Last = TBB, Unknown = -1 };
+  enum ThreaderType
```

Comment:
Weren't `enum`s now supposed to be `enum class`?

Reply:
Not uniformly.  

This one looks like it could be upon first glance.  Out of scope for this PR. Please make an issue and we'll update later.

Reply:
Out of scope for this PR.  See #1194